src/domain/app/layout/pageWrapper.module.scss

```diff
@@ -1,3 +1,7 @@
 .pageWrapper {
   display: grid;
 }
+
+.defaultContainer {
+  padding-bottom: 5rem;
```

Comment:
This put extra 3rem padding to bottom of container to separate 2rem of footer Koros, this can be reduce to 3rem when HDS Koros is in used